packages/core/src/components/button/_button-group.scss

```diff
@@ -148,7 +148,8 @@ Styleguide components.button-group.css
         @include pt-button-minimal-divider();
         display: inline-block;
         position: absolute;
-        top: 0;
+        top: 10%;
+        bottom: 10%;
```

Comment:
why percentage instead of pixel offset? i'd expect `top/bottom: $pt-grid-size` to look good in these cases.